projects/uitdatabank/docs/search-api/sorting.md

```diff
@@ -6,6 +6,7 @@ The UiTdatabank Search API provides several options to sort events, places and o
 * `created`
 * `distance`
 * `modified`
+* `completeness`
```

Comment:
The sorting parameters are also ordered alphabetically, so I would move this to line 5.